components/bps-tools/plugins/examples/plugins/org.eclipse.bpel.extensionsample.model/src/org/eclipse/bpel/extensionsample/model/util/ExtensionSampleActivityDeserializer.java

```diff
@@ -1,89 +0,0 @@
-package org.eclipse.bpel.extensionsample.model.util;
```

Comment:
Shall we add add License header in these files ?

Reply:
The file is deleted and no longer in the code base.